com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -420,10 +420,12 @@ void SortCameras(Camera[] cameras)
 
             if (camera.targetTexture != null)
             {
-                desc.colorFormat = camera.targetTexture.descriptor.colorFormat;
-                desc.depthBufferBits = camera.targetTexture.descriptor.depthBufferBits;
-                desc.msaaSamples = camera.targetTexture.descriptor.msaaSamples;
-                desc.sRGB = camera.targetTexture.descriptor.sRGB;
+                desc = camera.targetTexture.descriptor;
```

Comment:
This will override all the data that has been written to desc so far (width and height).

Reply:
I noticed that the desc is actually already set to camera.targetTexture.descriptor at line 418 which means this operation is redundant and there wouldn't be any loss of data but maybe it's an error when not setting the width and height for the render texture when having a render target.